amqp/src/main/java/io/cloudevents/amqp/AmqpBinaryMessageReader.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2020-Present The CloudEvents Authors
```

Comment:
AFAIR, this should be 2018-Present, can you check the others too?

Reply:
ok, will change accordingly.. i was thinking the start date indicates the date file was created.

Reply:
It was agreed in past that we should use the same for all the files since the beginning of the project